pkgs/development/libraries/hamlib/default.nix

```diff
@@ -1,20 +1,50 @@
-{lib, stdenv, fetchurl, perl, python2, swig, gd, libxml2, tcl, libusb-compat-0_1, pkg-config,
- boost, libtool, perlPackages }:
+{ lib
+, stdenv
+, fetchurl
+, perl
+, swig
+, gd
+, python2
```

Comment:
Can we use python3 instead?

Reply:
Indeed, I've managed to get this building with `python3`.  Changes pushed.